test/python/compiler/test_transpiler.py

```diff
@@ -92,6 +92,14 @@ def _define(self):
         self._definition.cx(0, 1)
 
 
+def connected_qubits(physical: int, coupling_map: CouplingMap) -> set:
+    """Get the physical qubits that have a connection to this one in the coupling map."""
+    for component in coupling_map.connected_components():
+        if physical in (qubits := component.graph.nodes()):
+            return set(qubits)
```

Comment:
Not that efficiency matters much here but would it be better to do 
```suggestion
        if physical in (qubits := set(component.graph.nodes())):
            return qubits
```
so there is only a single iteration over the list?

Reply:
Haha, funnily enough I actually did it this way thinking roughly that on average over all calls, the constant factors associated with the unnecessary set constructions (for membership tests that fail) would outweigh the cost of the second iteration in success case.  I didn't bother to test, though.

Reply:
Heh, I guess at the end of the day it doesn't really matter. This is just test code so performance isn't critical and at most it's a list with like 30 entries.